### PR TITLE
gpmg has 100% shrapnel rng

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
the gpmg buff we've all been waiting for

## Why It's Good For The Game
i think this is a good idea

## Changelog
:cl:
balance: gpmg always shrapnels on hit
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
Based.

**Comment by Swagile** [comment]
april fools is over bro

**Comment by TGCMbetcher** [comment]
Now bad t60 users can be banned for griefing

**Comment by neohenry** [comment]
time for medics to take out more than 10 shrapnels from marines' chest

**Comment by Losenis** [comment]
30 damage per tile walked over having 20 shrap in your torso

**Comment by TGCMbetcher** [comment]
> 30 damage per tile walked over having 20 shrap in your torso

die in 5 steps from organ damage

**Comment by RyderFive** [comment]
Exactly what the game needs.

**Comment by bowlofcereal** [comment]
yes

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -476,6 +476,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	hud_state = "rifle_heavy"
 	damage = 20
 	penetration = 10
+	shrapnel_chance = 100
```
Comment (XSlayer300) [inline code/modules/projectiles/ammo_datums.dm:479]:
```suggestion
	shrapnel_chance = 0
```

**Comment by SplinterGP** [comment]
this is truly a wonder from balancing PR's

File: code/modules/projectiles/ammo_datums.dm (hunk 1 above)
Comment (Naaanii) [inline code/modules/projectiles/ammo_datums.dm:479]:
```suggestion
	shrapnel_chance = 100
	sundering = 1
```
Because why not?

**Review by Naaanii — CHANGES_REQUESTED** [review]

**Comment by TiviPlus** [comment]
Funny

**Comment by Naaanii** [comment]
> Funny

https://github.com/tgstation/TerraGov-Marine-Corps/pull/6484#issuecomment-812854790

**Pull request closed without merging**